README.md

```diff
@@ -1,6 +1,9 @@
 # Collection for Event-Driven Ansible
 
-This collection contains event source plugins, event filters and example rulebooks to be used with [ansible-rulebook](https://ansible-rulebook.readthedocs.io/en/stable/).
+This collection contains event source plugins, event filters and example rulebooks to be used with [eda-server](https://github.com/ansible/eda-server).
```

Comment:
The plugins and filters can be used with ansible-rulebook CLI, so I think the most accurate statement would be:
```suggestion
This collection contains event source plugins, event filters and example rulebooks to be used with [ansible-rulebook](https://ansible-rulebook.readthedocs.io/en/stable/) or [eda-server](https://github.com/ansible/eda-server).
```